app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
```

Comment:
Using `argparse` from Python standard library can greatly improve the readability of your code. However, it is not mandatory, since according to README, using `sys.argv` is enough.